frontend/src/components/MeetingRoom/VideoTileCanvas/VideoTileCanvas.tsx

```diff
@@ -74,8 +75,12 @@ const VideoTileCanvas = ({
     wrapRef,
   });
 
-  // Height is 100vh - toolbar height (80px) - 24px wrapper margin
-  const heightClass = '@apply h-[calc(100vh_-_104px)]';
+  // Height is 100vh - toolbar height (80px) and header height (96px) - 24px wrapper margin on mobile
+  // Height is 100vh - toolbar height (80px) - 24px wrapper margin on desktop
+  const heightClass = isMobile()
+    ? '@apply h-[calc(100vh_-_200px)]'
+    : '@apply h-[calc(100vh_-_104px)]';
+
```

Comment:
Note: this was needed to accommodate for the space that the header adds but given that it is only used on mobile, this had to go two different paths.